### PR TITLE
Update location of tsp-ops

### PR DESCRIPTION
**Pull request by tbekolay**
See details and discuss at https://github.com/nengo/nengo-edge-hw/pull/134 (any changes done there will be applied here afterward).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -98,10 +98,11 @@ jobs:
           path: nengo-edge
       - uses: actions/checkout@v4
         with:
-          repository: nengo/tsp-ops
-          path: tsp-ops
+          repository: nengo/tsp_chip_hw
+          path: tsp_chip_hw
+          sparse-checkout: tsp_ops
```
Comment (drasmuss) [inline .github/workflows/ci.yml:103]: Do we actually need this repo? I think this might be a historical artifact, like in nengo-edge-models.

Comment (tbekolay) [reply .github/workflows/ci.yml:103]: Hm, good question ... Some tests do use `nengo_edge_hw`, but they're not using the TSP right now. I'm not sure if you can import `nengo_edge_hw` with `tsp_ops` not installed?

Comment (drasmuss) [reply .github/workflows/ci.yml:103]: You should be able to yeah, we make the `tsp-ops` imports conditional in `nengo-edge-hw`.

Comment (tbekolay) [reply .github/workflows/ci.yml:103]: Cool, I'll switch this to not cloning `tsp_ops` then.

**Pull request merged**